help/mention-a-user-or-group.md

```diff
@@ -73,13 +73,13 @@ streams](/help/stream-notifications).
 ## Mention everyone on a stream
 
 You can mention everyone on a stream with the `@**all**` mention (or,
-equivalently, `@**everyone**` or `@**stream**`). It's best to use these mentions
+equivalently, `@**everyone**` or `@**channel**`). It's best to use these mentions
 only when your message requires timely attention from many stream subscribers.
 An organization can be configured to
 [restrict](/help/restrict-wildcard-mentions) wildcard mentions in large streams.
 
 Users can disable notifications for `@**all**`, `@**everyone**`, and
-`@**stream**` mentions by:
+`@**channel**` mentions by:
```

Comment:
I think we can defer this change to the PR that migrates the strings. 

A good way to think about it is that a stopping point for this PR would be that is has the API changes and everything that we can merge todayish that makes `@channel` work, but doesn't regress anything that works today -- so the new `channel` would just be undocumented for now, and the string changes PR would include the hunks from this that update documentation and typeahead.

Reply:
Cool. I've removed the help center and deprecation of "stream" changes from this PR. And will plan on including them in the main user-facing string changes/PR.